Sources/NIOHTTP2/HTTP2Error.swift

```diff
@@ -1572,7 +1574,9 @@ public enum NIOHTTP2Errors {
     /// meaningfully be `Equatable`, so they aren't. There's also no additional location information: that's
     /// provided by the base error.
     public struct StreamError: Error {
-        private final class Storage {
+        private final class Storage: @unchecked Sendable {
```

Comment:
The storage isn't sendable, but `StreamError` is `@unchecked Sendable` because it does the CoW dance when modifying storage.